src/modules/pageRendering.js

```diff
@@ -0,0 +1,84 @@
+import { pokemonAPI } from './APIs.js';
+
+export default async (listData) => {
+  const pokemonList = document.getElementById('pokemonList');
+  pokemonList.innerHTML = '';
+  // to get the pokemon main object that has name & url
+  const response = await fetch(pokemonAPI);
+  listData = await response.json();
+  listData = listData.results;
+
+  // console.log(listData);
+
+  // to sort the array elements
+  const sortedDataList = [...listData];
+  sortedDataList.sort((a, b) => b.score - a.score);
+  // console.log(sortedDataList);
+  const objects = sortedDataList;
+  //   console.log(objects[0]);
+  // to loop through 18 elements of the array
+  for (let i = 0; i < 18; i += 1) {
+    const li = document.createElement('li');
+    const pokemonPopup = document.querySelector('.popup');
+    const commentBtn = document.querySelectorAll('.commentPopup');
+
+    const object = objects[i];
+    // to fetch the pokemon Image
+    const response = fetch(object.url);
+    const pokeObject = response.json();
+    // console.log(pokeObject);
+
+    // to get the image of pokemon
+    //  console.log(pokeObject.sprites.other.dream_world.front_default);
```

Comment:
- [ ] Please remove any unused code from your codebase for a cleaner and more optimized environment. 🙏 see more tips [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.).